src/Logging/Logging.Abstractions/src/EventId.cs

```diff
@@ -67,7 +67,7 @@ public override string ToString()
         }
 
         /// <summary>
-        /// Indicates whether the current object is equal to another object of the same type.
+        /// Indicates whether the current object is equal to another object of the same type. Two events are equal if they the same Id.
```

Comment:
`.. if they **have** the same id.` (Note that `id` is lowercase. Or change it to a `<ref>` thingy if you want to specify the `Id` property.)